projects/kserve/testing/config.yaml

```diff
@@ -125,6 +125,9 @@ ci_presets:
 
   # ---
 
+  4min:
+    tests.e2e.llm_load_test.duration: 240
+
```

Comment:
I might be missing something but can you please explain why you added 4 min test but afaik we are not using it right?

Reply:
FYI @npalaska (and @dagrayvid), the settings in this section (`.presets`) don't necessarily have to be used right away. The presets can be passed as parameter when launching the test, eg I may want to add a `20min` preset, and launch it manually every once in a while to ensure that it gives the expected results
```
test rhoai-light kserve 20min
```